front/src/Phaser/Game/GameScene.ts

```diff
@@ -952,6 +952,23 @@ ${escapedMessage}
                 tags: this.connection ? this.connection.getAllTags() : []
             })
         }));
+        this.iframeSubscriptionList.push(iframeListener.changeTileStream.subscribe((eventTiles) => {
+            for (const eventTile of eventTiles) {
+                const layer = this.gameMap.findPhaserLayer(eventTile.layer);
+                if ( layer ) {
+                    const tileIndex = this.getIndexForTileType(eventTile.tile);
+                    if ( tileIndex ) {
+                        this.gameMap.putTileInFlatLayer(tileIndex, eventTile.x, eventTile.y, eventTile.layer);
+                        const tile = layer.putTileAt(tileIndex, eventTile.x, eventTile.y);
+                        for (const property of this.gameMap.getTilesetProperties()[tileIndex]) {
+                            if ( property.name === "collides" ) {
+                                tile.setCollision(true);
+                            }
+                        }
+                    }
+                }
```

Comment:
This should be a function put in GameMap: `putTile`

(and `GameMap.putTileInFlatLayer` should be private)